### PR TITLE
Embed code into docker image

### PR DESCRIPTION
**Pull request by timlinux**
We need this for rancher catalogue - the code must ship with the image....

### CODE REVIEW

**Comment by timlinux** [comment]
Backport to master

File: Dockerfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ RUN apt-get -y install osm2pgsql
 ADD requirements.txt /requirements.txt
 RUN pip install -r requirements.txt
 
+ADD reporter /reporter
+
 ADD server.py /server.py
 # we will use a volume rather
 #ADD reporter /reporter
```
Comment (Gustry) [inline Dockerfile]: This line was already here

File: docker-compose.yml [hunk 2]
```diff
@@ -9,7 +12,7 @@ db:
 web:
   #image: kartoza/osm-reporter
   build: .
-  dockerfile: Dockerfile
+  dockerfile: Dockerfile.dev
```
Comment (Gustry) [inline docker-compose.yml]: Do you mean the not dev dockerfile is not used anymore?

Comment (timlinux) [reply docker-compose.yml]: The dev docker would be used for local development and the Dockerfile for pushing images to docker hub for use by the catalogue which cannot do local builds.

**Pull request merged**